js/interactive-guides/step-content.js

```diff
@@ -16,54 +16,108 @@ var stepContent = (function() {
 
   var setSteps = function(steps) {
     _steps = steps;
+    __createLinksToParentSteps();    
+  };
+
+  // Create link to parent step for each section so it can load the parent step when clicked on
+  var __createLinksToParentSteps = function() {
+    // Make links to parents
+    for(var i = 0; i < _steps.length; i++){
+      var step = _steps[i];
+      if(step.sections){
+        for(var j=0; j<step.sections.length; j++){
+          step.sections[j].parent = step;
+        }
+      }
+    }
   };
 
   var getCurrentStepName = function() {
     return currentStepName;
   };
 
-  // Hide the previous selected step content by looking for data-step attribute with the step name in it
+  // Hide the previous selected step content by looking for data-step attributes with the data-step not equal to the current step name
   var __hideContents = function() {
-    var stepToBeHidden = $("[data-step=" + currentStepName + "]");
-    stepToBeHidden.addClass("hidden");
+    var stepsToBeHidden = $("[data-step][data-step!=" + currentStepName + "]");
+    stepsToBeHidden.addClass("hidden");
   };
 
-  var __updateTitle = function(title) {
-    $(ID.blueprintTitle).html(title);
-    $(ID.blueprintTitle).attr('aria-label', title);
+  // Append the step title
+  var __addTitle = function(step) {
+    if(!step.title){
+      return;
+    }
+    var newTitle = $("<div class='title'></div>");
+    newTitle.attr('data-step', step.name);
+    newTitle.html(step.title);
+    $("#contentContainer").append(newTitle);
```

Comment:
do we need to add aria-label like the old codes?

Reply:
Thanks, added it.